projenrc/benchmark-test.ts

```diff
@@ -132,7 +132,7 @@ export class BenchmarkTest {
           },
           {
             name: 'Authenticate Via OIDC Role',
-            if: `github.event.repository.fork == false && github.ref == 'refs/heads/main'`,
+            if: `github.event.repository.fork == false && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/maintenance)`,
```

Comment:
Should this be a `startWith`?

Reply:
Wouldn't a push to the maintenance branch be exactly `refs/heads/maintenance`?

Reply:
Oh nvm, maintenance branches are named with the version number...